src/routes/safe/components/Balances/SendModal/screens/ReviewTx/index.jsx

```diff
@@ -93,9 +96,10 @@ const ReviewTx = ({
 
   const submitTx = async () => {
     const web3 = getWeb3()
+    const ready = await onboardUser()
```

Comment:
isn't one check here is enough?

Reply:
Yes, you are right!

Reply:
I think we can check this only in `createTransaction` method so we don't have to call this in every place we want to execute a transaction